reference/json-rpc/filecoin.md

```diff
@@ -2,14 +2,41 @@
 
 ## FilecoinAddressToEthAddress
 
-FilecoinAddressToEthAddress converts an f410 or f0 Filecoin Address to an EthAddress
+FilecoinAddressToEthAddress converts any Filecoin address to an EthAddress.
+
+This method supports all Filecoin address types:
+
+- "f0" and "f4" addresses: Converted directly.
```

Comment:
I checked other docs, let's keep the format the same when quoting addresses. 
- "f0" -> `f0` 
- "f1", "f2", and "f3" -> `f1/f2/f3`
Can you please update them all?